includes/class-activitypub.php

```diff
@@ -107,6 +107,7 @@ public static function render_json_template( $template ) {
 
 		if ( \is_author() ) {
 			$json_template = ACTIVITYPUB_PLUGIN_DIR . '/templates/author-json.php';
+			$template      = ACTIVITYPUB_PLUGIN_DIR . '/templates/author-json.php';
```

Comment:
This ends up enabling unsigned get requests for actors

Reply:
I thought about having them available but returning only a limited set of data then? (See some lines below)

Wasn't sure about the current best practice.

Reply:
I believe the current best practice is outlined [here](https://swicg.github.io/activitypub-http-signature/#instance-actor):
> [...] servers with authorized fetch enabled often use an [instance actor](https://seb.jambor.dev/posts/understanding-activitypub-part-4-threads/#the-instance-actor) to sign object fetches. **This is generally a "server-level" actor, separate from any normal user, that doesn't require an HTTP Signature to be fetched**. This breaks the loop of fetching each actor back and forth to validate their signatures.